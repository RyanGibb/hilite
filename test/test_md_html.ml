let md = {|
```ocaml
let html = "<h1>Hello World!</h1>"
```
|}

let test_transform () =
  let omd_t = Omd.of_string md |> Hilite.Md.transform in
  Format.printf "%s" (Omd.to_html omd_t)

let () = test_transform ()
