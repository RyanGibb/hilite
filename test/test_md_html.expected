<pre><code><span class='ocaml-keyword'>let</span><span class='ocaml-source'> </span><span class='ocaml-entity-name-function-binding'>html</span><span class='ocaml-source'> </span><span class='ocaml-keyword-operator'>=</span><span class='ocaml-source'> </span><span class='ocaml-string-quoted-double'>&quot;</span><span class='ocaml-string-quoted-double'>&lt;h1&gt;Hello World!&lt;/h1&gt;</span><span class='ocaml-string-quoted-double'>&quot;</span><span class='ocaml-source'>
</span></code></pre>